### ISSUE TITLE
UI Control enableModelRunning

### ISSUE THREAD

**Issue by llahiru** [fiebrink1/wekimini#9]
Hi all,

I am trying to enable a selected model using the following code in Processing. I am not sure what's wrong but it keeps giving me the error:

> Model numbers must be between 1 and # models for OSC message /wekinator/control/enableModelRecording

I have more than five models in my project.

```
OscMessage msg2 = new OscMessage("/wekinator/control/enableModelRunning");
msg2.add(1);
msg2.add(2);
```

I do not get the error if I add only one message ex: msg2.add(1), but it doesn't enable the model either.

I would really appreciate any help with this.

Thanks!

**Comment by fiebrink1** [comment]
Thanks! I'll look into that.

**Comment by llahiru** [comment]
I tried to debug the code and it seems like the option is available only for Supervised systems. Mine is a DTW. Is there a way to enable or disable model for a DTW system?

```
    //modelNum between 1 and numOutputs
    public void setModelRunEnabled(int modelNum, boolean enableRun) {
        if (checkEnabled() && isSupervised) {
            supervisedController.setModelRunEnabled(modelNum, enableRun);
        }
    }
```

Thanks!

**Comment by llahiru** [comment]
Okay, my bad. I just realised that there are no models in DTW, only gesture types. Is there a way to enable/disable (that play icon in front of each gesture) using OSC messages?
Thanks!

**Comment by fiebrink1** [comment]
Not at the moment, though that's a good idea for a feature to add!

**Comment by llahiru** [comment]
Yeah, it would be great to have that. For the time being, I added a function in **DtwModel.java** to fire a Property Change to which I have a listener in **DtwLearningRow.java**. Seems to be working :)
